Atos20/Crate
# Test branch

PR Template:
What’s this PR do?  
  - tests `user, crates & crates` reducers

Where should the reviewer start?  
 - `code/web/src/modules/crate/Item.js`
 - `code/web/src/modules/crate/api/crate_state.test.js`
 - `code/web/src/modules/user/api/user_actions.test.js `

How should this be manually tested?  
 
cd  into the web folder  and run `npm test`

Any background context you want to provide?  

- All test from the reducers should pass

- I started testing the `user actions file` but they became quite the challenge, so I decided to go back to them if we have extra time. I want to focus on making the API request to the server

What are the relevant tickets?  
- closes #37 
- closes #36
- closes #35
Screenshots (if appropriate)  
 
![Screen Shot 2020-12-10 at 4 30 14 PM](https://user-images.githubusercontent.com/56229864/101842312-fd450900-3b04-11eb-9abb-85660a1b7b00.png)

Questions: 
I noticed I squashed only 4 commits when I should have squashed all of them. I will make sure to do a proper squashing moving forward.
